src/watertap_contrib/reflo/costing/solar/photovoltaic.py

```diff
@@ -92,6 +92,7 @@ def build_photovoltaic_cost_param_block(blk):
 )
 def cost_pv(blk):
 
+    blk.costing_package.has_electricity_generation = True
```

Comment:
Just a comment on PV costs: it would be totally fair to cost PV out on a $/kW_dc basis. I remember using numbers on the order of $1000 - $2000/kWdc

Reply:
I think this is how we do it already?